test_requirements.txt

```diff
@@ -2,7 +2,6 @@
 colour-runner==0.0.4
 coverage==3.7.1
 dj-database-url==0.3.0
-django==1.7.4
```

Comment:
I think that would be safe to do. Django is very good at not breaking things with micro releases, and these requirements are for a dev environment anyway.